nixos/modules/services/monitoring/smartd.nix

```diff
@@ -239,11 +239,7 @@ in
 
     systemd.services.smartd = {
       description = "S.M.A.R.T. Daemon";
-
       wantedBy = [ "multi-user.target" ];
-
-      path = [ pkgs.nettools ]; # for hostname and dnsdomanname calls in smartd
```

Comment:
Is nettools no longer necessary?

Reply:
IIRC smartmontools wants `inetutils`, which has been added to the package directly through the `scriptpath` option.

Reply:
IIRC, both `nettools` and `inetutils` have the `hostname` and `dnsdomainname` commands, but with different command-line options. Are there particular flags that smartmontools is using @pacien?

Reply:
Last time I checked (a few months ago), smartmontools' `smartd_warning.sh` seemed to work correctly only with the inetutils implementation.